frontend/.env

```
@@ -0,0 +1 @@
+﻿REACT_APP_API_URL=http://localhost:3010/api
```

dotenv-linter (3.3.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)